wrapper/libsrc/vmaf_lib.cpp

```diff
@@ -0,0 +1,779 @@
+/**
```

Comment:
Similar comment to the other file. Much of the this code is a copy-paste. I would split this to a call to existing interface and then population of the new data structure.